apps/frontend/src/utilities/export_util.ts

```diff
@@ -1,10 +1,18 @@
-import ZipFile from 'adm-zip';
 import {saveAs} from 'file-saver';
+let zip: any;
```

Comment:
Unfortunately necessary for now due to dynamic import.  The blocker is top level await.  I'm working on getting that enabled.